packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -146,6 +165,7 @@ export class DialogBase extends React.Component<IDialogProps, {}> {
           onDismiss={onDismiss ? onDismiss : dialogContentProps!.onDismiss}
           className={contentClassName || dialogContentProps!.className}
           {...dialogContentProps}
+          styles={mergedDialogContentStyles}
```

Comment:
instead of providing styling overrides, just pass along a prop like `isDragging`. That way it can be used in styling.